datawave-stack/values.yaml

```diff
@@ -2,5 +2,27 @@
 # Default Values for Docker Registry #
 ######################################
 
+global:
+  hadoopConfigMap: dwv-hadoop-config
+  accumuloConfigMap: dwv-accumulo-config
+  hdfsWebhdfsEnabled: false
+
 dockerRegistry:
   url: ghcr.io/nationalsecurityagency/
+charts:
+  zookeeper:
+    enabled: false
```

Comment:
These are disabled by defualt, which is a change to how startLocalTest.sh was before, and changes the behavior on first checkout.

Reply:
agreed. I made the change locally late last Friday during my testing. helps if I commit and push.